detectors/node/opentelemetry-resource-detector-docker/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@opentelemetry/resource-detector-docker",
+  "version": "1.0.0",
```

Comment:
not sure we can release directly as 1.0 ? waiting for other maintainers input on this

Reply:
I would definitely prefer to start with 0.1.0